README.md

```diff
@@ -1,8 +1,50 @@
-# Final-Project-Template
+# The Rise of EVs and their Impact on the Environment
 <!-- Edit the title above with your project title -->
 
 ## Project Overview
 
+### Topic:
+The rapid increase in electric vehicles (EV) available on the road has offered a very promising solution to global warming efforts around the world. However, understanding the impact of EVs on the environment starts with the full lifecycle starting from the manufacturing to usage on the road and inevitably disposal. This topic is important because it seems that people ar shifting over to EVs for everyday commuting due to the financial benefits and reliefs that may come in the form of gas and maintenance. This paired with the rising concern in the environment and global warming brings it fairly high on the relevance charts.
+
+### Project Questions and Potential Answer Formats:
+
+#### What is the current state of EV adoption around the world, and which geographic regions are experiencing the most significant growth?
+> Including visuals, such as a bar graph or a heat map, can show sales and growth trends in different regions.
+
+#### How do greenhouse gas emissions through the lifespan of an EV compare to the traditional gas-powered combustion engine?
```

Comment:
This is a really great point to make. I would consider adding another dataset for combustion engines and there impacts to draw a good comparision between the two.